src/components/ContentHighlights/ContentHighlightToast.jsx

```diff
@@ -0,0 +1,34 @@
+import { Toast } from '@edx/paragon';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+const ContentHighlightToast = ({ toastText }) => {
+  /* Toast visibility state initially set to false to ensure the toast's
+  fade-in animation occurs on mount once `showToast` set to true. */
+  const [showToast, isToastShown] = useState(false);
```

Comment:
nit: in this renaming, looks like you lost the `set` in the setter function variable's name :) e.g., `showToast` and `setShowToast` / `isToastShown` and `setIsToastShown`.